hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaAction.java

```diff
@@ -112,6 +117,20 @@
     @JoinColumn(name = "rollout", updatable = false, foreignKey = @ForeignKey(value = ConstraintMode.CONSTRAINT, name = "fk_action_rollout"))
     private JpaRollout rollout;
 
+    @Column(name = "maintenance_cron_schedule")
+    private String maintenanceSchedule;
```

Comment:
Can you add `javax.validation` to the fields like min or max length? You can checkout the constraints and their definition like on the JpaTarget how it's used and for documentation purposes the constraints are public available in the API of the entities like `Target#ADDRESS_MAX_SIZE`